docs/feature-gates.md

```diff
@@ -81,6 +82,15 @@ and will not implement Cluster IP functionality as expected.
 When using the OVS built-in kernel module (which is the most common case), your
 kernel version must be >= 4.6 (as opposed to >= 4.4 without this feature).
 
+### AntreaProxyFull
+
+`AntreaProxyFull` enables full Service support in AntreaProxy. After enabling this
+feature, all type of Services can be accessed from outside the cluster.
```

Comment:
if we do decide that a Feature Gate is appropriate for this, this should explain the interaction with kube-proxy IMO

Reply:
`AntreaProxyFull` enables full Service support in AntreaProxy. After enabling this
feature, without KubeProxy, NodePort/LoadBalancer can be accessed from outside the
cluster, and ClusterIP can be accessed from host.